workshops/multiplayer/README.md

```diff
@@ -0,0 +1,286 @@
+---
+name: Node Intro + Express
+description: Make a simple webserver!
+author: 'Cedric Hutchings'
+group: start
+order: 9
+---
+
+# Express Webserver
+
+## What can you learn in this workshop?
+A webserver is a little bit of software that lets you send websites -- and all of the files that they need -- to anyone who connects to your server.
+I'll admit, making a webserver seems a little prosaic.
+It's not even immediately evident *why* you'd want to bother making one.
+In truth, I'm making this workshop as the beginning of a tutorial series I'm running on how to make a webgame with networked multiplayer, but there really are plenty of other reasons to make your own webserver.
```

Comment:
"really" is unnecessary and a bit strange sounding (maybe cuz my socal english) ur call

Reply:
interesting, definitely doesn't sound off to me but I'll go ahead and remove it.